src/themes/jekyll/_includes/components/js-include.html

```diff
@@ -0,0 +1,8 @@
+<script src="app.js"></script>
+
+{% if page.show_page_js %}
+  <details open>
+    <summary>This Page</summary>
```

Comment:
"This Page" is a bit unclear as to what it's referring to.

Maybe being more explicit. "JavaScript in this Page"?